broker/src/bin/bench.rs

```diff
@@ -0,0 +1,175 @@
+use std::sync::atomic::{AtomicU64, Ordering};
+use std::sync::Arc;
+use std::time::{Duration, Instant};
+
+use clap::Parser;
+use neon_broker::neon_broker_proto::neon_broker_client::NeonBrokerClient;
```

Comment:
`crate` won't work here, as this is binary crate, but I reduced that.

Reply:
Still do not get it why we cant call it just broker. Imports still look like that `use neon_broker::neon_broker_proto::`. Why not just broker::proto?

Reply:
It would be possible if lib exported these names directly in global namespace. However, there is
```
// Code generated by protobuf.
pub mod neon_broker_proto {
    include!("../proto/neon_broker.rs");
}
```

I find it useful that all generated code is put into separate module, helps differentiate with manually written one. I also saw the same structure in all tonic apps/examples I encountered.

Reply:
I'm ok with separate module, was just wondering why we cant call the module just `proto`. I see you made this change under different discussion item. Thanks!